docs/openDAQ-streaming.md

```diff
@@ -392,15 +392,20 @@ There are some example of signal descriptions in a separate document.
       <contains at least one signal member description>
     },
     "tableId": <string>,
+    ["valueIndex": <uint64>],
     ["relatedSignals": [
         { "type" : <relation type>, "signalId" : <id of the related signal> } 
       ]
-    ]
+    ],
     ["interpretation": {}]
   }
 }
 ~~~~
 
+- `tableId`: Id of the table the signal belongs to
+- `valueIndex`: Optional, and relevant only if domain signal of the table follows an implicit linear rule.

```

Comment:
Same comment as PR 22. Suggest changin "an implicit linear rule" to "an implicit rule such as a implicit linear time signal"

Reply:
Will do!